packages/web-components/src/components/back-to-top/back-to-top.ts

```diff
@@ -169,6 +169,7 @@ class C4DBackToTop extends HostListenerMixin(StableSelectorMixin(LitElement)) {
     const { backToTopAssistiveText, _handleOnClick: handleOnClick } = this;
     return html`
       <button
+        part="back-to-top__button"
```

Comment:
```suggestion
        part="button"
```

Reply:
Created a new commit using this cleaner nomenclature. I'm going to update my other PRs as well.